state/subnets.go

```diff
@@ -56,8 +63,10 @@ type subnetDoc struct {
 	AvailabilityZone  string `bson:"availabilityzone,omitempty"`
 	// TODO: add IsPublic to SubnetArgs, add an IsPublic method and add
 	// IsPublic to migration import/export.
-	IsPublic  bool   `bson:"is-public,omitempty"`
-	SpaceName string `bson:"space-name,omitempty"`
+	IsPublic         bool   `bson:"is-public,omitempty"`
+	SpaceName        string `bson:"space-name,omitempty"`
+	FanLocalUnderlay string `bson:"fan-local-underlay,omitempty"`
```

Comment:
We'd have to remember that any fields we add that are stored in the DB will need to be migrated.